homework/shelkovich_ilya/arrayService/src/main/java/com/tutrit/java/quickstart/service/ArrayService.java

```diff
@@ -0,0 +1,51 @@
+package com.tutrit.java.quickstart.service;
+
+import java.util.Arrays;
+import java.util.concurrent.ThreadLocalRandom;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ArrayService {
+
+    Logger log = LoggerFactory.getLogger(ArrayService.class);
+
+    public int[] createArray(int size) {
+        return new int[size];
+    }
+
+    public int[] fillRandomly(int[] array) {
+        for (var i = 0; i < array.length; i++) {
+            array[i] = ThreadLocalRandom.current().nextInt(-1, 101);
+        }
+        return array;
+    }
+
+    public void printArray(int[] array) {
+        for (int value : array) {
+            log.info("{}", value);
+        }
+    }
+
+    public int sum(int[] array) {
+        var sum = 0;
+        for (int j : array) {
+            sum = j + sum;
+        }
+        return sum;
+    }
+
+    public double avg(int[] array) {
+        return (double) sum(array) / array.length;
```

Comment:
I would rather cast array.length that result. Guess why? try tests with random values.

Reply:
why should I cast sum of elements in array to array length? Can't get it

Reply:
I can't get it what I did wrong

Reply:
I can't understand how to fix it

Reply:
resolved